docs/Dev_eXo_Addons.rst

```diff
@@ -1327,6 +1327,67 @@ To add an event listener using one of listed events above, you can add the follo
           <type>org.example.CustomEventListener</type>
         </component-plugin>
       </external-component-plugins>
+      
+      
+.. _PLFDevGuide.eXoAdd-ons.eXoOnlyOfficeConnector:
+
+==============================
+OnlyOffice connector for eXo
+==============================
+
+With :ref:`OnlyOffice connector <OnlyOffice>`, it is possible to add new document types to the
+form ``New document`` by using the :ref:`extension mechanism <PLFDevGuide.eXoAdd-ons.PortalExtension.Mechanism>`.
```

Comment:
Can we add a link to the "New document" chapter in the user guide ?

Reply:
OK

Reply:
done